### ISSUE TITLE
Lapis cannot be used in Studio Widget with Team Create enabled

### ISSUE THREAD

**Issue by Halferi** [nezuo/lapis#47]
Lapis is currently dependent on `game:BindToClose` during module import and this event seems to not work in projects with Team Create enabled.

**Comment by nezuo** [comment]
I'm not sure I understand. Lapis works fine in Team Create enabled projects. Are you trying to use Lapis in a Studio plugin?

**Comment by Halferi** [comment]
Apologies for not describing it well in the description. Yes, this is related to Studio plugins. Trying to import the libarary from a Studio plugin will cause the library to break, as the `game:BindToClose` is used on import and it's not supported in Studio plugins.

Not sure if there are any suitable workarounds to support Studio plugins without `game:BindToClose`, but thought it would be good to bring it up.

**Comment by nezuo** [comment]
Yeah, I will probably disable that in Studio plugins. I just wasn't sure how Team Create was relevant to the issue.
